akd/src/tree_node.rs

```diff
@@ -884,13 +854,13 @@ pub(crate) fn optional_child_state_hash(input: &Option<TreeNode>) -> Digest {
 }
 
 /// Create an empty root node.
-pub async fn create_empty_root<S: Database + Sync + Send>(
+pub(crate) async fn create_empty_root<S: Database + Sync + Send>(
     storage: &StorageManager<S>,
     ep: Option<u64>,
-    least_descendant_ep: Option<u64>,
+    smallest_descendant_ep: Option<u64>,
 ) -> Result<TreeNode, StorageError> {
-    // Empty root hash is the same as empty node hash
-    let empty_root_hash = crate::utils::empty_node_hash_no_label();
```

Comment:
nit: Can we leave this as a ref into this own crate? Example

```rust
crate::hash::hash(&crate::EMPTY_VALUE)
```

That way we don't have explicit calls into `akd_core` which imo is cleaner. If you see any other fully referenced calls to `akd_core` you could definitely relate them to `crate::` prefixes! We should have re-imported the entire core lib. 

The reason: If anyone looks at the `akd` crate source, or say examples, comments, if everything is `crate::` referenced they know they don't need to explicitly pull sub-dependencies (like `akd_core`) which helps mitigate versioning compat problems downstream.

Reply:
Sounds good. Replaced a bunch of instances in `akd` from `akd_core::hash` to `crate::hash`